pkg/controller/servicebindingrequest/binder.go

```diff
@@ -46,6 +45,7 @@ type Binder struct {
 }
 
 var EmptyApplicationSelectorErr = errors.New("application ResourceRef or MatchLabel not found")
+var ApplicationNotFound = errors.New("Application is already deleted")
```

Comment:
From:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings
`Error strings should not be capitalized (unless beginning with proper nouns or acronyms) ...`